packages/api-client-core/src/operationBuilders.ts

```diff
@@ -131,15 +131,18 @@ export const actionOperation = (
   modelSelectionField: string,
   variables: VariablesOptions,
   options?: SelectionOptions | null,
-  namespace?: string | null
+  namespace?: string | null,
+  isBulkAction?: boolean | null,
```

Comment:
This was added since bulk operations have a `results` field instead of `result`. We could avoid this if we generated a different default selection based on whether or not an action has a return type. Not sure if that's needed since the `actionRunner` already takes an `isBulkAction` arg already.